e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/validation/multiple-contract-policy-validation-maximum-buyer-will-owe.spec.js

```diff
@@ -0,0 +1,154 @@
+import { submitButton } from '../../../../../pages/shared';
+import { typeOfPolicyPage, multipleContractPolicyPage } from '../../../../../pages/insurance/policy-and-export';
+import partials from '../../../../../partials';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { APPLICATION, FIELD_IDS, ROUTES } from '../../../../../../../constants';
+import getReferenceNumber from '../../../../../helpers/get-reference-number';
+import checkText from '../../../../../helpers/check-text';
+
+const { taskList } = partials.insurancePartials;
+
+const multiplePolicyFieldId = FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.POLICY_TYPE;
+const multiplePolicyField = typeOfPolicyPage[multiplePolicyFieldId].multiple;
+
+const { INSURANCE } = ROUTES;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        MULTIPLE: { MAXIMUM_BUYER_WILL_OWE },
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        MULTIPLE: CONTRACT_ERROR_MESSAGES,
+      },
+    },
+  },
+} = ERROR_MESSAGES;
+
+context('Insurance - Policy and exports - Multiple contract policy page - form validation - maximum buyer will owe', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
+
+    multiplePolicyField.input().click();
+    submitButton().click();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+      const expectedUrl = `${Cypress.config('baseUrl')}${INSURANCE.ROOT}/${referenceNumber}${INSURANCE.POLICY_AND_EXPORTS.MULTIPLE_CONTRACT_POLICY}`;
```

Comment:
can we add a tech debt ticket to dereference all of these? its in nearly all files

Reply:
There is a ticket already to move this to a cypress command 😄 Will update the ticket to also do the destructuring https://ukef-dtfs.atlassian.net/browse/EMS-775